content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -209,6 +210,24 @@ kubelet:65536:7208960
 [CVE-2021-25741]: https://github.com/kubernetes/kubernetes/issues/104980
 [shadow-utils]: https://github.com/shadow-maint/shadow
 
+## ID count for each of Pods
+Starting with Kubernetes v1.33, the ID count for each of Pods can be set in
+[`KubeletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/).
+
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+userNamespaces:
+  idsPerPod: 1048576
+```
+
+The value of `idsPerPod` (uint32) must be a multiple of 65536.
+The default value is 65536.
+Changing the value may require recreating all containers on the node.
```

Comment:
nit: It seems vague on when it is required or not. I'd say it is never _required_

I'd just says what happens: the setting applies to containers created after the setting was applied, running containers are not affected (rephrase this to something appropriate)

Reply:
done